.gitignore

```diff
@@ -19,6 +19,7 @@ Dockerrun.aws.json
 audit-report.*
 .DS_Store
 .env
+.env.ocp
```

Comment:
What exactly is this file for? Shouldn't have to do with HCAP-394.

Reply:
Another .env I use for tunneling into the openshift db. I got tired of removing it from my commits.

Reply:
Maybe we should go with .env.* ?